### PR TITLE
Support APIExport identity in the Syncer virtual workspace

### PR DESCRIPTION
**Pull request by davidfestal**
Support `APIExport` identity in the Syncer virtual workspace.

This is a pre-quisite for PR #927

### CODE REVIEW

File: pkg/virtual/framework/forwardingregistry/rest_test.go [hunk 1]
```diff
@@ -59,9 +59,10 @@ func (mcg *mockedClusterClient) Cluster(cluster logicalcluster.Name) dynamic.Int
 	return mcg.client
 }
 
-func newStorage(t *testing.T, clusterClient dynamic.ClusterInterface, patchConflictRetryBackoff *wait.Backoff) customresource.CustomResourceStorage {
-	groupResource := schema.GroupResource{Group: "mygroup.example.com", Resource: "noxus"}
-	gvr := groupResource.WithVersion("v1beta1")
+var noxusGVR schema.GroupVersionResource = schema.GroupVersionResource{Group: "mygroup.example.com", Resource: "noxus", Version: "v1beta1"}
```
Comment (sttts) [inline pkg/virtual/framework/forwardingregistry/rest_test.go:62]: noxus is back 😱

Comment (davidfestal) [reply pkg/virtual/framework/forwardingregistry/rest_test.go:62]: was already there :-)
Copied this from somewhere initially. No idea where it comes from.

File: pkg/virtual/syncer/controllers/apireconciler/apireconciler_controller.go [hunk 2]
```diff
@@ -181,6 +206,54 @@ func (c *APIReconciler) enqueueNegotiatedAPIResource(obj interface{}) {
 	}
 }
 
+func apiExportGroupResources(apiExport *apisv1alpha1.APIExport) map[schema.GroupResource]interface{} {
+	grs := map[schema.GroupResource]interface{}{}
+	for _, resourceSchema := range apiExport.Spec.LatestResourceSchemas {
+		parts := strings.SplitN(resourceSchema, ".", 3)
+		if len(parts) != 3 {
+			klog.Errorf("invalid LatestResourceSchemas list element on APIExport %s|%s: %s", logicalcluster.From(apiExport), apiExport.Name, resourceSchema)
```
Comment (sttts) [inline pkg/virtual/syncer/controllers/apireconciler/apireconciler_controller.go:214]: continue?

Comment (davidfestal) [reply pkg/virtual/syncer/controllers/apireconciler/apireconciler_controller.go:214]: Oh, yes, sure !

Comment (davidfestal) [reply pkg/virtual/syncer/controllers/apireconciler/apireconciler_controller.go:214]: let me rebase and force-push for this fix.

Comment (davidfestal) [reply pkg/virtual/syncer/controllers/apireconciler/apireconciler_controller.go:214]: Done

File: pkg/virtual/framework/forwardingregistry/rest_test.go (hunk 1 above)
Comment (ncdc) [reply pkg/virtual/framework/forwardingregistry/rest_test.go:62]: I think it's a deads2k special 😄

**Review by sttts — APPROVED** [review]

**Pull request merged**